app/locations/controllers.test.js

```diff
@@ -214,5 +204,25 @@ describe('Locations controllers', function () {
         )
       })
     })
+
+    context('when there is an API error', function () {
+      it('should fail gracefully for regions API', async function () {
+        req.session.user.permissions = ['allocation:create']
+        req.services.referenceData.getRegions = sinon.fake.returns(
+          Promise.reject(new Error())
+        )
+        await proxiedController.locations(req, res, nextSpy)
```

Comment:
I also made a change to remove the need to use proxyquire here as we weren't mocking any dependencies of this module so it was unnecessary. You can change this to the following once rebase:
```suggestion
        await locationsController(req, res, nextSpy)
```